<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntegralCommand</class>
 <widget class="QWidget" name="IntegralCommand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QWidget" name="additionalContentWidget" native="true"/>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="readWriteLabel">
     <property name="text">
      <string>Read Only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
